tests/countries/test_anguilla.py

```diff
@@ -0,0 +1,337 @@
+#  holidays
+#  --------
+#  A fast, efficient Python library for generating country, province and state
+#  specific sets of holidays on the fly. It aims to make determining whether a
+#  specific date is a holiday as fast and flexible as possible.
+#
+#  Authors: Vacanza Team and individual contributors (see CONTRIBUTORS file)
+#           dr-prodigy <dr.prodigy.github@gmail.com> (c) 2017-2023
+#           ryanss <ryanssdev@icloud.com> (c) 2014-2017
+#  Website: https://github.com/vacanza/holidays
+#  License: MIT (see LICENSE file)
+
+from unittest import TestCase
+
+from holidays.countries.anguilla import Anguilla, AI, AIA
+from tests.common import CommonCountryTests
+
+
+class TestAnguilla(CommonCountryTests, TestCase):
+    @classmethod
+    def setUpClass(cls):
+        years = range(2001, 2050)
+        super().setUpClass(Anguilla, years=years, years_non_observed=years)
+
+    def test_country_aliases(self):
+        self.assertAliases(Anguilla, AI, AIA)
+
+    def test_special_holidays(self):
+        self.assertHoliday("2011-04-29", "2012-06-04", "2025-02-28")
```

Comment:
I've also added our new QE II's Mournign Day to the list
```suggestion
    def test_special_holiday(self):
        self.assertHolidayName("Royal Wedding of Prince William & Kate Middleton", "2011-04-29")
        self.assertHolidayName(
            "Diamond Jubilee Celebration of Her Majesty The Queen", "2012-06-04"
        )
        self.assertHolidayName(
            "Mourning the Death of Her Majesty The Queen Elizabeth II", "2022-09-19"
        )
        self.assertHolidayName("Special Public Holiday", "2025-02-28")
```